### DISCUSSION TITLE
Curso basico de Gps con Laravel

### DISCUSSION THREAD

**Discussion by ysernah** [eusonlito/GPS-Tracker#60] [Ideas]
Hola amigo excelente desarrollo mi sugerencia seria que realices un curso basico con laravel para poder conectar algun gps del mercado obiamente un curso simple porque un curso del software que desarrollaste seria muy largo el curso saludos

**Comment by eusonlito** [comment]
Ya me gustaría disponer de tiempo para eso! 😅
